sky/packages/sky/lib/src/gestures/arena.dart

```diff
@@ -45,6 +45,7 @@ class GestureArenaEntry {
 class _GestureArenaState {
   final List<GestureArenaMember> members = new List<GestureArenaMember>();
   bool isOpen = true;
+  bool isHeld = false;
```

Comment:
Can we use an enum instead of a collection of bools?  For example, is it valid to hold an arena that is no longer open?